spec/search_builders/adv_search_builder_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+RSpec.describe AdvSearchBuilder do
+  let(:scope) do
+    double(blacklight_config: CatalogController.blacklight_config,
+           current_ability: ability)
+  end
+  let(:user) { create(:user) }
+  let(:ability) { ::Ability.new(user) }
+  let(:access) { :read }
+  let(:builder) { described_class.new(scope).with_access(access) }
+
+  it "can be instantiated" do
+    expect(builder).to be_instance_of(described_class)
+  end
+
+  describe ".default_processor_chain" do
+    subject { described_class.default_processor_chain }
+
+    let(:expected_default_processor_chain) do
+      # Yes there's a duplicate for add_access_controls_to_solr_params; but that does not appear to
+      # be causing a problem like the duplication and order of the now removed additional
+      # :add_advanced_parse_q_to_solr, :add_advanced_search_to_solr filters.  Those existed in their
+      # current position and at the end of the array.
+      #
+      # When we had those duplicates, the :add_advanced_parse_q_to_solr obliterated the join logic
```

Comment:
@orangewolf I'm drawing attention to this particular add.  The breadcrumbs are almost adequate and I think what I'm seeing in test on Hyrax 5 Upgrade will be ok.  But this is a subtle bug and we're about to go on break so I'm trying to make sure to provide insight/guidance for possible troubleshooting.

<img width="1126" alt="Screenshot 2023-12-21 at 4 45 00 PM" src="https://github.com/samvera/hyku/assets/2130/599f2757-c474-4d65-aec8-e3c0f8e64717">

Reply:
I updated in the Hyrax 5 upgrade: 1929072d38a41b4420bd1cb936692c4fc6848e97

It may be alright to back port.  We'll know when we enter into more testing/QA on that branch.